src/jit/codegenxarch.cpp

```diff
@@ -7587,8 +7588,14 @@ void CodeGen::genRemoveAlignmentAfterCall(GenTreeCall* call, unsigned bias)
         SubtractNestedAlignment(padStkAlign);
     }
 #else  // UNIX_X86_ABI
-    assert(bias == 0);
+    if (bias != 0)
+    {
+        inst_RV_IV(INS_add, REG_SPBASE, bias, EA_PTRSIZE);
```

Comment:
I'd prefer the previous one using `genAdjustSP` as bias is for caller/callee argument but this function is for stack alignment.